client/dashboard/sites/logs-activity/dataviews/index.tsx

```diff
@@ -147,7 +148,37 @@ function SiteActivityLogsDataViews( {
 			url.searchParams.delete( 'search' );
 		}
 		window.history.replaceState( null, '', url.pathname + url.search );
-
+		if ( perPageChanged ) {
+			recordTracksEvent( 'calypso_dashboard_activity_logs_per_page_changed', {
```

Comment:
```suggestion
			recordTracksEvent( 'calypso_dashboard_site_activity_logs_per_page_changed', {
```

Worthh making it consistent. There's many more of such instances in this PR.

Reply:
Good catch!